README.md

```diff
@@ -9,41 +9,21 @@ It includes a status reducer that lets you track the status of your async action
 This package provides an action creator utility, that together with the provided middleware will make it very easy to create side effects for your actions.
 This setup will automatically execute your side effect thunk and dispatch success or error actions when the thunk succeeds or fails, respectively.
 
-Example:
-```js
-// src/actions/userActions.js
-import { createThunk } from '@rootstrap/redux-tools'
-
-export const getProfile = createThunk(
-  'GET_PROFILE',
-  userService.getProfile,
-);
-```
-
-If you need to access your store, or dispatch extra actions from your thunk, you can use `dispatch` and `getState` as parameters your thunk.
+`createThunk` receives the action names prefix as the first argument and the async thunk as the second one.
```

Comment:
```suggestion
`createThunk` receives the action's name prefix as the first argument and the async thunk as the second one.
```

Reply:
It's the prefix for the multiple action names